core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -319,41 +279,47 @@ wasi_addr_ip_to_bh_ip_addr_buffer(__wasi_addr_ip_t *addr,
     }
 }
 
+bool
```

Comment:
Please note this function returns bool, and the converted clock value should be passed through the `out`parameter. In your case, you return the converted value from the function.